sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql

```diff
@@ -0,0 +1,83 @@
+WITH activity_stream AS (
+  SELECT
+      DATE(submission_timestamp) AS submission_date,
+      normalized_country_code  AS country,
+      COUNT(*) AS as_sessions,
+      COUNT(DISTINCT client_id ) AS as_clients
+  FROM `mozdata.activity_stream.sessions`
+  WHERE
+    (metadata.geo.country ) IN (
+      'AU',
+      'BR',
+      'CA',
+      'DE',
+      'ES',
+      'FR',
+      'GB',
+      'IN',
+      'IT',
+      'JP',
+      'MX',
+      'US'
+      )
+    AND ((page ) IN ('about:home', 'about:newtab')
+    AND (user_prefs ) >= 258)
+    AND `mozfun`.norm.browser_version_info(version).major_version >= 91
+    AND DATE(submission_timestamp) = @submission_date
+  GROUP BY
+      1, 2
```

Comment:
The guidance in the [DTMO SQL style guide](https://docs.telemetry.mozilla.org/concepts/sql_style) is to be explicit with this kind of thing

```suggestion
  GROUP BY
      submission_date,
      country
```